DataTypes.php

```diff
@@ -9,12 +9,12 @@
 
 namespace DataTypes;
 
-if ( defined( 'DataTypes_VERSION' ) ) {
+if ( defined( 'DATATYPES_VERSION' ) ) {
 	// Do not initialize more than once.
 	return 1;
 }
 
-define( 'DataTypes_VERSION', '1.0.0' );
+define( 'DATATYPES_VERSION', '1.0.0' );
```

Comment:
This is a breaking change. I don't think this alone justifies a 2.0.0 release. See my suggestion in #70 to solve this issue.